pages/1.10/release-notes/1.10.5/index.md

```diff
@@ -0,0 +1,250 @@
+---
+layout: layout.pug
+navigationTitle:  Release Notes for 1.10.5
+title: Release Notes for 1.10.5
+menuWeight: 0
+excerpt:
+---
+
+These are the release notes for DC/OS 1.10.5.
+
+[button color="purple" href="https://downloads.dcos.io/dcos/stable/1.10.5/dcos_generate_config.sh"]Download DC/OS Open Source[/button]
+
+
+# Issues Fixed in DC/OS 1.10.5
+
+- COPS-1963 - The agent admin router no longer exposes the default Nginx page.
+- DCOS-13824 - Removed list of cluster packages from cloud config and turned it into a build artifact that is downloaded during node setup.
+- DCOS-18784 - The DC/OS UI now shows the correct number of instances in a pod.
+- DCOS-19008 - Master certificates now include exhibitor_address if exhibitor_address is an IP address. [enterprise type="inline" size="small"/]
+- DCOS-19534 - The agent admin router no longer serves the default error page when accessing document root.
```

Comment:
DCOS-19534 tracks the ENG effort while solving/addressing COPS-1963. I am not sure we want to mention it twice.